Sources/BrowserServicesKit/UserScript/UserScript.swift

```diff
@@ -33,6 +34,19 @@ public protocol UserScript: WKScriptMessageHandler {
 
 extension UserScript {
```

Comment:
Similar to the comment above, we can add a default implementation for the new `requiresRunInPageContentWorld` property:

```
public var requiresRunInPageContentWorld: Bool {
    return false
}
```